### PR TITLE
Dyn 2579 - Ignoring test that left web browser open

### PR DESCRIPTION
**Pull request by alfredo-pozo**
Please Note:
1. Before submitting the PR, please review [How to Contribute to Dynamo](https://github.com/DynamoDS/Dynamo/blob/master/CONTRIBUTING.md)
2. Dynamo Team will meet 1x a month to review PRs found on Github (Issues will be handled separately)
3. PRs will be reviewed from oldest to newest
4. If a reviewed PR requires changes by the owner, the owner of the PR has 30 days to respond. If the PR has seen no activity by the next session, it will be either closed by the team or depending on its utility will be taken over by someone on the team
5. PRs should use either Dynamo's default PR template or [one of these other template options](https://github.com/DynamoDS/Dynamo/wiki/Choosing-a-Pull-Request-Template) in order to be considered for review.
6. PRs that do not have one of the Dynamo PR templates completely filled out with all declarations satisfied will not be reviewed by the Dynamo team.
7. PRs made to the `DynamoRevit` repo will need to be cherry-picked into all the DynamoRevit Release branches that Dynamo supports. Contributors will be responsible for cherry-picking their reviewed commits to the other branches after a `LGTM` label is added to the PR.

### Purpose

Ignore tests that are leaving web browser open after the execution

### Declarations

Check these if you believe they are true

- [x] The codebase is in a better state after this PR
- [x] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [x] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [x] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

**Comment by alfredo-pozo** [comment]
https://jira.autodesk.com/browse/DYN-2579

**Comment by alfredo-pozo** [comment]
The evidence of the test on SelfServe where we can see that those tests are being ignored https://master-5.jenkins.autodesk.com/job/Dynamo/job/DynamoSelfServe/job/master/26/testReport/Dynamo.Tests/CrashReportingTests/

**Comment by QilongTang** [comment]
1. Seem something wrong with your indent setting? Saw couple PRs from you always have extra indents in the new lines added. Please fix it.
2. I think the ignore setting can be merged with the test tag, no?

File: test/DynamoCoreWpfTests/CrashReportingTests.cs [hunk 1]
```diff
@@ -43,6 +43,7 @@ at System.Windows.Application.RunInternal(Window window)
             at DynamoSandbox.DynamoCoreSetup.RunApplication(Application app)";
 
         [Test]
+		[Ignore("Test ignored because the web browser is not closed after the test execution")]
```
Comment (QilongTang) [inline test/DynamoCoreWpfTests/CrashReportingTests.cs:46]: hmm.. Is this a setting thing or encoding setting issue?

**Comment by QilongTang** [comment]
LGTM

**Pull request merged**